src/libs/Middleware/Reauthentication.js

```diff
@@ -39,7 +39,9 @@ function Reauthentication(response, request, isFromSequentialQueue) {
                         return data;
                     }
 
-                    request.resolve(data);
+                    if (request.resolve) {
```

Comment:
NAB, might want to add a comment (or create an issue to clean up later) as this code (and other things in `/Network` referencing `request.resolve`) can be removed once we are fully migrated off of the `DeprecatedAPI`.

Reply:
https://github.com/Expensify/Expensify/issues/212275